clusters/app.ci/art/admin_art.yaml

```diff
@@ -0,0 +1,130 @@
+---
+- kind: Group
```

Comment:
I think this should not be a single-item YAML list :)

Reply:
_facepalm_ . Thanks, @petr-muller !